cypress/support/commands.js

```diff
@@ -23,3 +23,19 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('login', (username = 'user675', password = 'qwertY123!') => {
+  cy.request({
+  method: 'POST',
+  url: 'https://demoqa.com/Account/v1/Login',
+  body: {
+  userName: username,
+  password: password
+  },
+  }).then((response) => {
+  cy.setCookie('token', response.body.token);
+  cy.setCookie('expires', response.body.expires);
+  cy.setCookie('userID', response.body.userId);
+  cy.setCookie('userName', response.body.username);
+  });
+  })
```

Comment:
It's a good practice to have a newline at the end of the file. Please add a newline at the end of the file. 😊